### ISSUE TITLE
Move the package directory to a separate repo

### ISSUE THREAD

**Issue by Looooong** [alelievr/Mixture#54]
In essence, this repo is a development project that includes in Mixture package. I would love to have the package directory moved to a separated repo and included in this repo as a git submodule.

The reason behind this is that I cannot use registry package because I want to make a few modifications. On the other hand, I also cannot use local package because I cannot share the modification with my colleague. In my use case, I want to fork ONLY the package as a repo and can easily add this forked repo to Unity Package Manager using git URL. By doing so, I can collab with my colleague on both the Unity project along with the forked repo.

Basically, you can setup your repos like one of the following:
+ Use `Mixture` repo for dev project and use `Mixture-package` repo for package.
+ Use `Mixture` repo for package and use `Mixture-dev` for dev project.
+ Use `Mixture` repo for package and move the dev project info a orphaned branch under `Mixture` repo. This is what I have done for [my Unity package](https://github.com/Looooong/Unity-SRP-VXGI) where the dev project is moved to branch [`legacy`](https://github.com/Looooong/Unity-SRP-VXGI/tree/legacy) and is also [distributed in Releases as demo project](https://github.com/Looooong/Unity-SRP-VXGI/releases/tag/v0.0.1)

**Comment by Jha-2022** [comment]
Check it once and if any needed please inform
